CMake/platforms/windows.cmake

```diff
@@ -20,3 +20,5 @@ endif()
 if(MINGW_CROSS)
   list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/mingw")
 endif()
+
+set(FULL_RANGE_CONTROLLER ON)

```

Comment:
Your control scheme requires an LT/L2 button on the controller in order to access all in-game functions, correct? I don't think we should be making the assumption at build time that LT/L2 will be available on controllers used by Windows, Linux, and Mac players.

Reply:
I don't see them playing with a controller at all xD but if they do that why wouldn't they use a Xbox/PS controller? Even 8bitdo controllers have L2/R2.

Reply:
My understanding is that the control scheme that was developed for DevilutionX was designed specifically to support 12-button gamepads. I don't see a compelling reason to restrict platforms that support those types of controllers.

I'm especially uncompelled if your the justification is "why wouldn't they use a different controller"? It doesn't really matter why they wouldn't, but rather whether there exists a player that wouldn't. And the only answer I can give you is "maybe yes".

Reply:
Also potentially relevant to this discussion. #2965

Reply:
TBH my initial idea was to only add this controller config to the Xbox port but I was criticized but doing that. So I have added the option for doing it to any system you want through CMAKE. I also agree with the idea that we need a major change and allow customization by the user but until there we are not allowing Xbox player to have a better controller scheme. That is my opinion regarding all this discussion

Reply:
I have an idea for having multiple predefined control schemes. I'll push it as soon as I get it done.

Reply:
I know @NiteKat at least has a gamepad with very few buttons connected to his computer :D